code/modules/mob/living/living_defense.dm

```diff
@@ -254,22 +254,23 @@
 /mob/living/proc/grabbedby(mob/living/user, supress_message = FALSE)
 	if(user == src || anchored || !isturf(user.loc))
 		return FALSE
+
 	if(!user.pulling || user.pulling != src)
 		user.start_pulling(src, supress_message = supress_message)
 		return
-	// This line arbitrarily prevents any non-carbon from upgrading grabs
-	if(!iscarbon(user))
```

Comment:
how sure are you that this isn't here for a reason

Reply:
I linked issue, where I have discussed it with @MrMelbert